packages/beacon-node/src/util/sszBytes.ts

```diff
@@ -66,16 +70,38 @@ export function getBlockRootFromAttestationSerialized(data: Uint8Array): BlockRo
 }
 
 /**
- * Extract attestation data base64 from attestation serialized bytes.
+ * Extract attestation data key from an attestation Uint8Array in order to index gossip queue and cache later in SeenAttestationDatas
+ */
+export function getSeenAttDataKey(forkSeq: ForkSeq, data: Uint8Array): SeenAttDataKey | null {
+  return forkSeq >= ForkSeq.electra ? getSeenAttDataKeyElectra(data) : getSeenAttDataKeyPhase0(data);
+}
+
+/**
+ * Extract attestation data + committeeBits base64 from electra attestation serialized bytes.
+ * Return null if data is not long enough to extract attestation data.
+ */
+export function getSeenAttDataKeyElectra(electraAttData: Uint8Array): AttDataCommitteeBitsBase64 | null {
```

Comment:
It's passing in a `Attestation` and not `AttestationData`. `electraAtt` is more appropriate than `electraAttData`